app/imports/ui/pages/AddSmartContract.jsx

```diff
@@ -9,19 +9,22 @@ import { SmartContracts } from '../../api/smartContract/SmartContract';
 
 // Create a schema to specify the structure of the data to appear in the form.
 const contractSchema = new SimpleSchema({
-  name: String,
-  email: String,
-  phoneNumber: String,
-  role: {
+  homeownerName: String,
+  homeownerEmail: String,
+  homeownerPhoneNumber: String,
+  tenetName: String,
+  tenetEmail: String,
+  tenetPhoneNumber: String,
+  tenetStance: {
     type: String,
-    allowedValues: ['Tenet', 'Homeowner'],
-    defaultValue: 'Tenet',
+    allowedValues: ['I do not agree to the terms and conditions', 'I agree to the terms and conditions', ''],
```

Comment:
a more common approach  is to not store the actual text in the DB. the actual phrase is displayed in the UI , and the backend stores a simple string "agreed" or null (null for missing agreement).

Reply:
but it's ok to merge and possibly adjust later if we have time.